src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2117,7 +2117,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         double width = cols * actualFontSize.X;
 
         // Reserve additional space if scrollbar is intended to be visible
-        if (scrollState == ScrollbarState::Visible)
+        if (scrollState == ScrollbarState::Visible || scrollState == ScrollbarState::Always)

```

Comment:
Would it be better to express these 3 occurrences as `scrollState != ScrollbarState::Hidden`?

Reply:
I'd vote for this, but not as a blocking request. I'll leave the PR open for a while just in case :smile:

Reply:
I had it that way originally...I can't remember why I changed it. I'll make that change.